data/data/gcp/network/internal-lb.tf

```diff
@@ -0,0 +1,36 @@
+resource "google_compute_health_check" "api_internal" {
+  name = "${var.cluster_id}-api-internal"
+
+  https_health_check {
```

Comment:
this needs a health check for 22623 `/healthz`

Reply:
https://github.com/openshift/installer/blob/11aa8211d062648b20842cf5a512aabdd95bb76e/data/data/aws/vpc/master-elb.tf#L102-L108

Reply:
I'm not sure we can do that. `health_checks - (Required) The list of HealthChecks for checking the health of the backend service. Currently at most one health check can be specified, and a health check is required.`

https://www.terraform.io/docs/providers/google/r/compute_region_backend_service.html#health_checks

When I tried more than one health check I got `Error: health_checks: attribute supports 1 item maximum, config has 2 declared`

```
diff --git a/data/data/gcp/network/internal-lb.tf b/data/data/gcp/network/internal-lb.tf
index afa055edd..999b1619f 100644
--- a/data/data/gcp/network/internal-lb.tf
+++ b/data/data/gcp/network/internal-lb.tf
@@ -7,6 +7,15 @@ resource "google_compute_health_check" "api_internal" {
   }
 }
 
+resource "google_compute_health_check" "api_internal_ign" {
+  name = "${var.cluster_id}-api-internal-ign"
+
+  https_health_check {
+    port         = 22623
+    request_path = "/healthz"
+  }
+}
+
 resource "google_compute_region_backend_service" "api_internal" {
   name = "${var.cluster_id}-api-internal"
 
@@ -22,7 +31,10 @@ resource "google_compute_region_backend_service" "api_internal" {
     }
   }
 
-  health_checks = [google_compute_health_check.api_internal.self_link]
+  health_checks = [
+    google_compute_health_check.api_internal.self_link,
+    google_compute_health_check.api_internal_ign.self_link
+  ]
 }
```

When I add more than one https_health_check to the google_global_health_check I got `Error: Too many https_health_check blocks`

```
diff --git a/data/data/gcp/network/internal-lb.tf b/data/data/gcp/network/internal-lb.tf
index afa055edd..14042a5b4 100644
--- a/data/data/gcp/network/internal-lb.tf
+++ b/data/data/gcp/network/internal-lb.tf
@@ -5,6 +5,11 @@ resource "google_compute_health_check" "api_internal" {
     port         = 6443
     request_path = "/readyz"
   }
+
+  https_health_check {
+    port         = 22623
+    request_path = "/healthz"
+  }
 }
```

Reply:
6443 is fine as only one health check is allowed.